R/downloadbuttons.R

```diff
@@ -0,0 +1,541 @@
+## MetaInsight - downloadable files for data uploading instructions - code.
+
+
+## Sizing functions for forest plots ##
+BayesPixels <- function(notrt, title=FALSE) {    # input is total number of treatments and whether title is included in plot
+  if (title==FALSE) {
+    if (notrt <= 25) {
+      height <- 420        # default for 25 or less treatments
+    } else {
+      height <- 15*(notrt-1) + 60
+    }
+  } else {
+    if (notrt <= 25) {
+      height <- 420 + 100        # default for 25 or less treatments
+    } else {
+      height <- 15*(notrt-1) + 60 + 100
+    }
+  }
+  return(height)
+}
+BayesInch <- function(notrt) {
+  if (notrt <= 25) {
+    height <- 6
+  } else {
+    height <- 6 + 0.2*(notrt-25)
+  }
+  return(height)
+}
+
+write_to_pdf_or_png <- function(file, type, renderFunction) {
+  if (tolower(type) == "pdf") {
+    pdf(file = file)
+  } else {
+    png(file = file)
+  }
+  renderFunction()
+  dev.off()
+}
+
+
+#########################
+### Tab 2 - Load data ###
+#########################
+
+create_raw_data_download_handlers <- function(input, output) {
+  
+  create_raw_data_download_handler <- function(filename, continuousFile, bindaryFile) {
```

Comment:
Is this a typo for binary? (And if not, is there a different word to use to avoid confusion?)

Reply:
Fixed